templates/www/cz/2019/devconfcz-2019-faq.html

```diff
@@ -186,6 +187,45 @@ <h2 id="speaker">{{ _('Speaker Questions') }}</h2>
 	</div>
 	</div>
 
+	<div class="clear"></div>
+
+	<h2 id="badge-flair">{{ _('Badge Flair Contest Questions') }}</h2>
+        <div class="toggle">
+        <a href="#" class="toggle-title"><strong>{{ _('Q: What is this badge flair contest anyway?') }}</strong><i class="fa fa-plus"></i></a>
+        <div class="toggle-content"><p>{{ _('The concept of the badge flair contest started at <a href="http://www.writethedocs.org/conf/prague/2018/badge-flair/">Write the Docs</a>, a conference and community for documentation-oriented tech folks who love to start fun and light-hearted traditions. The idea is simple: Decorate your badge, tweet a picture, and you might win a prize!') }}</p>
+	</div>
+	</div>
+
+	<div class="toggle">
+        <a href="#" class="toggle-title"><strong>{{ _('Q: Who can participate? Do I have to participate?') }}</strong><i class="fa fa-plus"></i></a>
+        <div class="toggle-content"><p>{{ _('All conference attendees and volunteers are eligible to participate. Organizing team members cannot enter the contest, although we do encourage you to lead by example and spread the word.') }}</p>
+          <p>{{ _('<u>The contest is optional.</u> Of course, you are free to use the decorating station even if you don’t want to participate, and decorate your badge for your own enjoyment!') }}</p>
```

Comment:
Don't use <u> tag. It's better to use <strong> / <b> tags.

Reply:
@jridky what do you mean by "/ tags"? I'm not familiar with this term, if you can give me exact syntax I'll happily replace it. Thanks!